app/frontend/src/utils/date.ts

```diff
@@ -2,10 +2,29 @@
 import { Timestamp } from "google-protobuf/google/protobuf/timestamp_pb";
 import moment from "moment";
 
+const MONTHS = [
+  "Jan",
+  "Feb",
+  "Mar",
+  "Apr",
+  "May",
+  "Jun",
+  "Jul",
+  "Aug",
+  "Sep",
+  "Oct",
+  "Nov",
+  "Dec",
+];
+
 export function formatDate(date: string, short: boolean = false): string {
   return moment(date, "YYYY-MM-DD").format(short ? "D MMM" : "D MMM YYYY");
 }
 
 export function timestamp2Date(timestamp: Timestamp.AsObject): Date {
   return new Date(Math.floor(timestamp.seconds * 1e3 + timestamp.nanos / 1e6));
 }
+
+export function monthYear(date: Date): string {
+  return `${MONTHS[(date.getMonth() + 1) % 12]} ${date.getFullYear()}`;
```

Comment:
I've been needing this in my PR too and I found that we can do it through the platform with the [`Intl.DateTimeFormat` API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat), so I don't think we need this. The advantage of using this is it's also configurable to format date for different language locales based on the user's language settings (with `navigator.language`). Both of these seem to have decent browser support so I think it's safe to go with it!